sorc/link_workflow.sh

```diff
@@ -333,33 +333,16 @@ fi
 
 # GDASApp
 if [[ -d "${HOMEgfs}/sorc/gdas.cd/build" ]]; then
-  declare -a JEDI_EXE=("fv3jedi_addincrement.x" \
-                       "fv3jedi_diffstates.x" \
-                       "fv3jedi_ensvariance.x" \
-                       "fv3jedi_hofx.x" \
-                       "fv3jedi_var.x" \
-                       "fv3jedi_convertincrement.x" \
-                       "fv3jedi_dirac.x" \
-                       "fv3jedi_error_covariance_training.x" \
-                       "fv3jedi_letkf.x" \
-                       "fv3jedi_convertstate.x" \
-                       "fv3jedi_eda.x" \
-                       "fv3jedi_forecast.x" \
+  declare -a JEDI_EXE=("gdas.x" \
```

Comment:
Probably need to add/remove these to the `.gitignore`

Reply:
[Looks like](https://github.com/danholdaway/global-workflow/blob/264f2fa2cbf2e6438113a8258c1468656c471a40/.gitignore#L20) it takes care of the whole directory in one go.